app/views/errors/internal_server_error_500.html.erb

```diff
@@ -0,0 +1,29 @@
+<% provide :head_tags do %>
+  <%= javascript_tag do %>
+    <%= render partial: 'page_not_found_404', formats: [:js] %>
```

Comment:
We probably need to rename this file to `_errors.js.erb` or something, since we're now using it on more than just the 404 page

Reply:
I renamed that include file.. and updated references to it.  Also removed the public 500.html file... good catch Josh.  You can test it locally now by updating that consider_all_requests_local var.